cmd/kubelet/app/options/options.go

```diff
@@ -290,14 +290,14 @@ func (f *KubeletFlags) AddFlags(mainfs *pflag.FlagSet) {
 		"On success, a kubeconfig file referencing the generated client certificate and key is written to the path specified by --kubeconfig. "+
 		"The client certificate and key file will be stored in the directory pointed by --cert-dir.")
 
-	fs.StringVar(&f.HostnameOverride, "hostname-override", f.HostnameOverride, "If non-empty, will use this string as identification instead of the actual hostname. If --cloud-provider is set, the cloud provider determines the name of the node (consult cloud provider documentation to determine if and how the hostname is used).")
+	fs.StringVar(&f.HostnameOverride, "hostname-override", f.HostnameOverride, "If non-empty, will use this string as identification instead of the actual hostname.")
```

Comment:
i think removing the mention of the `--cloud-provider` flag is good, should we maybe mention that running with an external cloud controller will cause the hostname to be provided by the ccm?

Reply:
Whatever the value of `--cloud-provider` is, the node name always takes the value of `os.Hostname()` or overridden with `--hostname-override`. There is no difference.

Reply:
I pushed new changes to remove unnecessary code which I forgot to remove in the previous commit. please review it again. thank you.

Reply:
thanks!